carbon/carbon.py

```diff
@@ -38,47 +34,21 @@ def _create_carbon_url(code, **carbon_options: str) -> str:
 
 def create_code_image(code: str, **kwargs: str) -> None:
     """Generate a beautiful Carbon code image"""
-    # options = Options()
-    # if not bool(kwargs.get("interactive", False)):
-    #     options.add_argument("--headless")
-
-    # service = (
-    #     Service(executable_path=kwargs["driver_path"])
-    #     if kwargs["driver_path"]
-    #     else Service()
-    # )
-
     destination = kwargs.get("destination", os.getcwd())
-    prefs = {"download.default_directory": destination}
-    # options.add_experimental_option("prefs", prefs)
+    headless = kwargs.get("headless", True)
```

Comment:
I agree, should be headless by default, bug what about controlling this with a DEBUG variable? so we can run it with headless=False if we want to see what it's doing in development / locally?

Reply:
Not sure, but I think it's related to the `--interactive` flag when it was still on Selenium. 
Need to figure out what to do with `--interactive` in cli.py

Reply:
What about the DEBUG env variable suggestion?